components/Form/MultiStepForm.js

```diff
@@ -9,74 +9,69 @@ import Alert from 'components/Alert/Alert';
 import ProgressIndicator from 'components/ProgressIndicator/ProgressIndicator';
 import styles from './MultiStepForm.css';
 
-export class MultiStepForm extends React.Component {
-  static propTypes = {
-    // initialValues must be object where entire form's shape is described
-    initialValues: object.isRequired,
+MultiStepForm.propTypes = {
+  // initialValues must be object where entire form's shape is described
+  initialValues: object.isRequired,
 
-    getErrorMessage: func.isRequired,
-    onEachStepSubmit: func,
-    onFinalSubmit: func.isRequired, // to be considered onSuccess
-    steps: arrayOf(validStep).isRequired,
-  };
+  getErrorMessage: func.isRequired,
+  onEachStepSubmit: func,
+  onFinalSubmit: func.isRequired, // to be considered onSuccess
+  steps: arrayOf(validStep).isRequired,
+};
 
-  static defaultProps = {
-    onEachStepSubmit: noop,
-  };
+MultiStepForm.defaultProps = {
+  onEachStepSubmit: noop,
+};
 
-  state = {
+export function MultiStepForm(props) {
```

Comment:
Can you destructure props at the top? It'll reduce LOC significantly.

Reply:
sure